posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12558 @@
+{
```

Comment:
<p>"Over the past few works,..."</p><p> </p><p>should be:</p><p> </p><p>"Over the past few weeks,..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>